src/main/java/com/additional/library/repository/BookRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.additional.library.repository;
+
+import com.additional.library.domain.Book;
+import org.springframework.data.repository.CrudRepository;
+import java.util.List;
+
+public interface BookRepository extends CrudRepository<Book, Integer> {
+    @Override
+    List<Book> findAll();
+
+    Book findById(Integer id);
+
+    @Override
+    Book save(Book book);
+
+    @Override
+    void delete(Integer id);
```

Comment:
w zadaniu nie ma mowu o usuwaniu czegokolwiek. Możesz to zrobić, jeśli chcesz, ale żeby nie została w repozytorium metoda, której nie użyjesz potem ;)